dali/python/backend_impl.cc

```diff
@@ -354,8 +354,9 @@ void ExposeTensorList(py::module &m) { // NOLINT
       R"code(
       List of tensors residing in the GPU memory.
       )code")
-    .def("asCPU", [](TensorList<GPUBackend> &t) -> TensorList<CPUBackend>* {
+    .def("as_cpu", [](TensorList<GPUBackend> &t) -> TensorList<CPUBackend>* {
           TensorList<CPUBackend> * ret = new TensorList<CPUBackend>();
+          ret->set_pinned(false);
```

Comment:
That's good as a temporary solution, but I think we might want to change the default to non-pinned. There's only a handful of cases where we _want_ pinned buffers.

Reply:
I agree